src/lib/customItems/customItems.ts

```diff
@@ -2766,3 +2765,236 @@ setCustomItem(
+	},
+	50_000_000
+);
+setCustomItem(
+	70_011,
+	'Dark Animica',
+	'Runite Ore',
+	{
+		customItemData: {
+			cantDropFromMysteryBoxes: true,
+			isSuperUntradeable: false
+		}
+	},
+	25_000
+);
+setCustomItem(
+	70_012,
+	'Tattered Tome',
+	'Magic Seed',
+	{
+		customItemData: {
+			cantDropFromMysteryBoxes: true,
+			isSuperUntradeable: true
+		}
+	},
+	25_000_000
+);
+setCustomItem(
+	70_013,
+	'Abyssal Tome',
+	'Book of law',
+	{
+		customItemData: {
+			cantDropFromMysteryBoxes: true,
+			isSuperUntradeable: true
+		},
+		equipment: {
+			attack_stab: 0,
+			attack_slash: 0,
+			attack_crush: 0,
+			attack_magic: 45,
+			attack_ranged: 0,
+
+			defence_stab: 0,
+			defence_slash: 0,
+			defence_crush: 0,
+			defence_magic: 35,
+			defence_ranged: 0,
+
+			melee_strength: 0,
+			ranged_strength: 0,
+			magic_damage: 2.5,
+			prayer: 0,
+			slot: EquipmentSlot.Shield,
+			requirements: { magic: 90 }
+		},
+		highalch: 50_000_000
+	},
+	50_000_000
+);
+setCustomItem(
+	70_014,
+	'Abyssal Gem',
+	'Zenyte',
+	{
+		customItemData: {
+			cantDropFromMysteryBoxes: true,
+			isSuperUntradeable: true
+		}
+	},
+	10_000_000
+);
+setCustomItem(
+	70_015,
+	'Abyssal Amulet',
+	'Amulet of Torture',
+	{
+		customItemData: {
+			cantDropFromMysteryBoxes: true,
+			isSuperUntradeable: true
+		},
+		equipment: {
+			attack_stab: 0,
+			attack_slash: 0,
+			attack_crush: 0,
+			attack_magic: 0,
+			attack_ranged: 0,
+
+			defence_stab: 0,
+			defence_slash: 0,
+			defence_crush: 0,
+			defence_magic: 0,
+			defence_ranged: 0,
+
+			melee_strength: 0,
+			ranged_strength: 0,
+			magic_damage: 0,
+			prayer: 0,
+			slot: EquipmentSlot.Neck,
+			requirements: null
+		},
+		highalch: 10_000_000
+	},
+	10_000_000
+);
+setCustomItem(
+	70_016,
+	'Spellbound Ring',
+	'Seers Ring',
+	{
+		customItemData: {
+			cantDropFromMysteryBoxes: true,
+			isSuperUntradeable: true
+		},
+		equipment: {
+			attack_stab: 0,
+			attack_slash: 0,
+			attack_crush: 0,
+			attack_magic: 20,
+			attack_ranged: 0,
+
+			defence_stab: 0,
+			defence_slash: 0,
+			defence_crush: 0,
+			defence_magic: 20,
+			defence_ranged: 0,
+
+			melee_strength: 0,
+			ranged_strength: 0,
+			magic_damage: 0,
+			prayer: 0,
+			slot: EquipmentSlot.Ring,
+			requirements: { magic: 90 }
+		},
+		highalch: 10_000_000
+	},
+	10_000_000
+);
+setCustomItem(
+	70_017,
+	'Spellbound Ring(i)',
+	'Seers Ring',
+	{
+		customItemData: {
+			cantDropFromMysteryBoxes: true,
+			isSuperUntradeable: true
+		},
+		equipment: {
+			attack_stab: 0,
+			attack_slash: 0,
+			attack_crush: 0,
+			attack_magic: 25,
+			attack_ranged: 0,
+
+			defence_stab: 0,
+			defence_slash: 0,
+			defence_crush: 0,
+			defence_magic: 25,
+			defence_ranged: 0,
+
+			melee_strength: 0,
+			ranged_strength: 0,
+			magic_damage: 0,
+			prayer: 0,
+			slot: EquipmentSlot.Ring,
+			requirements: { magic: 90 }
+		},
+		highalch: 10_000_000
+	},
+	10_000_000
+);
+setCustomItem(70_018, 'Grand Crystal Acorn', 'Coal', { tradeable: true }, 6000);
```

Comment:
at the very least, if not fixing all the incorrect casing in the whole PR, atleast fix it in this file where it defines the names.

Reply:
Fixed casing of all new items